lib/phlex/bucket.rb

```diff
@@ -1,24 +1,41 @@
 # frozen_string_literal: true
 
 module Phlex::Bucket
+	module Proxy
+		def method_missing(name, *args, **kwargs, &block)
+			if self.class.constants.include?(name) && self.class.const_get(name) && methods.include?(name)
+				public_send(name, *args, **kwargs, &block)
+			else
+				super
+			end
+		end
```

Comment:
Using `method_missing` here won’t be consistent when including modules that override other modules. The method you get will be based on which was autoloaded first, not which module has higher precedence. We should probably define these methods instead.